packager/config/nginx.conf.live.conf.template

```diff
@@ -6,7 +6,7 @@ vod_live_window_duration 0;
 vod_align_segments_to_key_frames on;
 vod_hls_absolute_master_urls off;
 vod_hls_absolute_index_urls off;
-vod_hls_output_id3_timestamps on;
+vod_hls_mpegts_output_id3_timestamps on;
 vod_dash_profiles urn:mpeg:dash:profile:isoff-live:2011;
```

Comment:
@erankor  Eran, did you change this? is this a breaking change?

Reply:
Look here: https://github.com/kaltura/nginx-vod-module/blob/master/CHANGELOG.md#20170813---hlsfmp4-support

Reply:
Yes, as written in the link @jessp01 pasted, I changed it when I added support for hls/fmp4. 3 configuration settings that were specific to hls/mpegts were renamed in order to make it clear that they don't apply to fmp4.